Tests/AmplitudeTests/AmplitudeTests.swift

```diff
@@ -324,6 +324,171 @@ final class AmplitudeTests: XCTestCase {
         ])
     }
 
+    func testMigrationToApiKeyAndInstanceNameStorage() throws {
+        let legacyUserId = "legacy-user-id"
+        let config = Configuration(
+            apiKey: "amp-migration-api-key",
+            // don't transfer any events
+            flushQueueSize: 1000,
+            flushIntervalMillis: 99999,
+            logLevel: LogLevelEnum.DEBUG,
+            defaultTracking: DefaultTrackingOptions.NONE
+        )
+
+        // Create storages using instance name only
+        let legacyEventStorage = PersistentStorage(storagePrefix: "storage-\(config.getNormalizeInstanceName())")
+        let legacyIdentityStorage = PersistentStorage(storagePrefix: "identify-\(config.getNormalizeInstanceName())")
+
+        print("Creating legacy Amplitude instance")
```

Comment:
nit: is it left intentional? Not sure if printing to console is a good practices in swift. 

The same for `print(legacyEventsString)` and `print(eventsString)` below.

Reply:
😅

Reply:
Fixed